src/TestFramework/Coverage/JUnit/JUnitTestCaseSorter.php

```diff
@@ -98,9 +156,11 @@ private function uniqueByTestFile(array $testLocations): array
         $uniqueTests = [];
 
         foreach ($testLocations as $testLocation) {
+            /** @var string $filePath */
             $filePath = $testLocation->getFilePath();
 
-            if (!array_key_exists($filePath, $usedFileNames)) {
+            // isset() is 20% faster than array_key_exists()
+            if (!isset($usedFileNames[$filePath])) {
```

Comment:
I thought this was no longer the case on PHP 7.3?

Reply:
I'll check if it so starting PHP 7.4. [In 7.3 it is like this.](https://gist.github.com/sanmai/d866617fed5e4aa1e3eee15c78abf39a)

Reply:
Fair enough, it's a hotpath anyway so I don't mind in this case